core/market/decentralized/src/negotiation/provider.rs

```diff
@@ -1,14 +1,16 @@
+// TODO: This is only temporary
+#![allow(dead_code)]
+
```

Comment:
Is there still any dead code in this file?

Reply:
Noope. :) Removing